README.md

```diff
@@ -100,6 +100,34 @@ export NEAR_ENV=mainnet
 
 ---
 
+### Custom RPC server selection
+You can set custom RPC server URL by setting this env variables:
+```bash
+NEAR_CLI_MAINNET_RPC_SERVER_URL
+NEAR_CLI_TESTNET_RPC_SERVER_URL
+NEAR_CLI_BETANET_RPC_SERVER_URL
+NEAR_CLI_GUILDNET_RPC_SERVER_URL
+NEAR_CLI_GUILDNET_RPC_SERVER_URL
```

Comment:
`NEAR_CLI_GUILDNET_RPC_SERVER_URL` is repeated twice

Reply:
Thank you, fixed!